ncard_app/templates/events/base.html

```diff
@@ -8,7 +8,7 @@
         <!-- Bootstrap JavaScript -->
         <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-select@1.14.0-beta2/dist/css/bootstrap-select.min.css" integrity="sha256-UqiEyrW1sB5d6ZDzcWXKfYCR4MKVYMEdXNjJde84cjc=" crossorigin="anonymous">
-        <link href="/static/table.css" rel="stylesheet">
+        <link href="/static/style.css" rel="stylesheet">
```

Comment:
You should use `{% load static %}` and `{% static 'style.css' %}` here, instead of in each form page